odl/operator/oputils.py

```diff
@@ -118,40 +118,91 @@ def matrix_representation(op):
     return matrix
 
 
-def power_method_opnorm(op, niter, xstart=None):
+def power_method_opnorm(op, xstart=None, maxiter=100, rtol=1e-05, atol=1e-08,
+                        callback=None):
     """Estimate the operator norm with the power method.
 
-    The operator norm ``||op(x)||`` is defined by as the smallest number such
-    that::
-
-        ||op(x)|| <= ||op|| ||x||
-
-    for all ``x``.
-
     Parameters
     ----------
     op : `Operator`
         Operator whose norm is to be estimated. If its `Operator.range`
         range does not coincide with its `Operator.domain`, an
         `Operator.adjoint` must be defined (which implies that the
         operator must be linear).
-    niter : positive `int`
-        Number of iterations to perform. If the domain and range of ``op``
-        do not match, it needs to be an even number.
     xstart : `Operator.domain` `element`, optional
         Starting point of the iteration. By default, the ``one``
         element of the `Operator.domain` is used.
+    maxiter : positive `int`, optional
+        Number of iterations to perform. If the domain and range of ``op``
+        do not match, it needs to be an even number. If None is given, iterate
+        untill convergence.
+    rtol : `float`, optional
+        Relative tolerance parameter (see Notes).
+    atol : `float`, optional
+        Absolute tolerance parameter (see Notes).
+    callback : `callable`, optional
+        Function called with the current iterate in each iteration.
 
     Returns
     -------
     est_opnorm : `float`
-        The estimated operator norm
+        The estimated operator norm of ``op``.
+
+    Examples
+    --------
+    Verify that the identity operator has norm 1
```

Comment:
Such lines should stop with "." or, my preference, ":" -- nothing at the end of the sentence looks ugly.